### ISSUE TITLE
LDAP Support

### ISSUE THREAD

**Issue by code1997** [hackerspace-bootstrap/strichliste#279]
Since most Hackerspaces have an LDAP / AD Server it would be awesome to be able to use this for the user backend. This should work as follows:
- User can login on strichliste with their LDAP credentials
- Once they login a strichliste user is created and the user may choose a pin
- Done, no more work for the admin.
- Overkill would be using LDAP Groups to auto define users as admins.

**Comment by schinken** [comment]
You're right! We're also using LDAP in our hackerspace. And thats a feature for strichliste2, which is currently under development!

Right now a simple script reading users from LDAP and using the REST Api to create them should be a good workaround

**Comment by schinken** [comment]
I'm currently working on strichliste2 and thinking about that feature. I guess there should be a combination of a import "cronjob" / console command and a direct ldap connection for authentication.

Still thinking about what one would want...
